<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderingParamsDialog</class>
 <widget class="QDialog" name="RenderingParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rendering Params</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>371</width>
     <height>362</height>
    </rect>
   </property>
   <widget class="QWidget" name="basicRenderingParams">
    <attribute name="title">
     <string>Rendering Params</string>
    </attribute>
    <widget class="Q3GroupBox" name="groupBox2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Colors</string>
     </property>
     <widget class="QComboBox" name="polygonMode">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>147</y>
        <width>79</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="colorSchemes">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>116</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="colorSchemesEditor">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>114</y>
        <width>20</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>32767</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Open color schemes editor</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="meshColorLabel">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>15</y>
        <width>63</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="colorMeshB">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>15</y>
        <width>20</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>32767</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="colorEdgeB">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>81</y>
        <width>20</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>32767</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="vertexColorLabel">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>48</y>
        <width>63</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="colorVertexB">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>48</y>
        <width>20</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>32767</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>81</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Edge Color</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>48</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Vertex Color</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>15</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Mesh Color</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>114</y>
        <width>64</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Color scheme</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="edgeColorLabel">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>81</y>
        <width>63</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>147</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Polygon Mode</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="Q3GroupBox" name="groupBox3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>174</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Feature Display</string>
     </property>
     <widget class="QLabel" name="textLabel1_6">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>100</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_4_3">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>72</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertices</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_4_2">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>44</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Edges</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_4">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>16</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Facets</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="vertexPainters">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>72</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="debugPainters">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="edgePainters">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>44</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="facetPainters">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>16</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="Q3GroupBox" name="groupBox1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>335</height>
      </rect>
     </property>
     <property name="title">
      <string>Display Modes</string>
     </property>
     <widget class="QCheckBox" name="cbSmoothShading">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>22</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Smooth Shading</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbCulling">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>46</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Culling</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbImposeVertices">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>70</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Superimpose Vertices</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbImposeEdges">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>94</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Superimpose Edges</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbLighting">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>118</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Lighting</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAntialiasing">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>142</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Antialiasing</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbUseNormals">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>166</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Normals</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbSmoothNormals">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>190</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Should use see facets</string>
      </property>
      <property name="text">
       <string>Smooth Normals</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbOverlay">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>220</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Overlay</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="edgeThickness">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSlider" name="alphaLevel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>106</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2_2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>300</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Alpha</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>270</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Edge Thickness</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="vertexThickness">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2_3">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>240</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertex Thickness</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>32767</width>
     <height>32767</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonApply">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>390</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>32767</width>
     <height>32767</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Apply</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>390</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>32767</width>
     <height>32767</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">RenderingParams.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorMeshB</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>changeMeshColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorEdgeB</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>changeEdgeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorVertexB</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>changeVertexColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>OnAlphaLevelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonApply</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorSchemesEditor</sender>
   <signal>clicked()</signal>
   <receiver>RenderingParamsDialog</receiver>
   <slot>OnEditColorSchemes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
